README.md

```diff
@@ -111,6 +132,84 @@ struct ExampleView: View {
 ```
 
 
+### Bulk Export of Historical Health Data
+
+The [`BulkHealthExporter`](https://swiftpackageindex.com/stanfordspezi/spezihealthkit/documentation/spezihealthkitbulkexport/bulkhealthexporter) enables export of historical HealthKit data through sessions that can process large amounts of data while maintaining memory efficiency and supporting resumption across app launches.
+
+First, configure the `BulkHealthExporter` in your `SpeziAppDelegate`:
+
+```swift
+import SpeziHealthKitBulkExport
```

Comment:
We might want to be consistent here to either always show an import or remove it from each example. Probably better to include it?